thirdparty_src_checksums.txt

```diff
@@ -32,15 +32,18 @@ af02b440ce3afe08db490eee1a685ca1e9d276d56a9ed2f2ea1ade2245426e67  libunwind-1.1a
 c7ccb735c37b4ec470f66a6c35fbae4f029c0f88038f6977180b1a8ddc255637  llvm-10.0.1.tar.gz
 8ad4ddbafac4f2c8f2ea523c2c4196f940e8e16f9e635210537582a48622a5d5  llvm-11.0.0.tar.gz
 9c7ad8e8ec77c5bde8eb4afa105a318fd1ded7dff3747d14f012758719d7171b  llvm-11.0.1.tar.gz
```

Comment:
Should we remove unused checksums from this file?

Reply:
I think it is fine to leave them here for now. We can bulk-remove them later when it becomes clear that we will never go back to certain old versions.